docs/_api/api-readme.md

```diff
@@ -1,35 +1,48 @@
 # LitElement API Documentation
 
-## Install lit-element
-
-```
-npm install lit-element
-```
-
 ## Modules
 
 ### [lit-element](/api/modules/_lit_element_.html)
 
-Extends [UpdatingElement](#lib-updating-element) to include lit-html templating. Users define a `render` method to provide an element template, which is called when required by `update`.
+The main LitElement module, which defines the [`LitElement`](/api/classes/_lit_element_.litelement.html) base class and related APIs.
+
+LitElement components can define a template and a set of observed properties. Changing an observed property triggers a re-render of the element.
 
-Import `LitElement` and `html` from this module to create a component:
+Import [`LitElement`](/api/classes/_lit_element_.litelement.html) and [`html`](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#html) from this module to create a component:
 
 ```js
-import { LitElement, html } from 'lit-element';
+import {LitElement, html} from 'lit-element';
 
 class MyElement extends LitElement {
+
+  // Declare observed properties
+  static get properties() {
+    return {
+      adjective: {}
+    }
+  }
+ 
+  constructor() {
+    this.adjective = 'awesome';
+  }
+
+  // Define the element's template
   render() {
-      return html`<p>your template here</p>`;
+    return html`<p>your ${adjective} template here</p>`;
   }
 }
 
 customElements.define('my-element', MyElement);
 ```
 
+`LitElement` extends [`UpdatingElement`](/api/classes/_lib_updating_element_.updatingelement.html) and adds lit-html templating.
+
 ### [lib/updating-element](/api/modules/_lib_updating_element_.html)
```

Comment:
Should we update the module name to `lit-element/lib/updating-element.js` since that's the import that needs to be used?

Reply:
SGTM